plutus-core/executables/plutus/AnyProgram/Compile.hs

```diff
@@ -131,7 +131,7 @@ compileProgram = curry $ \case
     (SUplc _ _, SPir SName _) -> throwingPIR "Cannot compile uplc to pir"
 
 embedProgram :: PLC.Program tyname name uni fun ann -> PIR.Program tyname name uni fun ann
-embedProgram (PLC.Program a v t) = PIR.Program a v $ embed t
+embedProgram (PLC.Program a v t) = PIR.Program a v $ embedTerm t
```

Comment:
`embed` was too nice of a name to steal for PIR stuff and I wanted to have it elsewhere. `embedTerm` now better reflects the nature of what's going on, especially given that `embedProgram` was already there.